### DISCUSSION TITLE
Supplementary Documentation

### DISCUSSION THREAD

**Discussion by spacejack** [MithrilJS/mithril.js#2933] [Meta/Feedback]
Regarding this repo: https://github.com/spacejack/mithril-docs-ex

This is a collection of additional documentation people might find useful after they go through the current set of docs on the site. These are to expand on topics that require more detail and nuance than you generally allow for in the API docs and could fill a similar role to Leo's old blog.

* Can we put these under MithrilJS somewhere so we can have long-term URLs?
* Where should we put them?
* Any edits or change suggestions welcome.
* Contributions are always welcome!
* Suggested additional topics: Look at Leo's blog and update for 1.x+

**Comment by thequailman** [comment]
These are good, though some of them are mostly in the docs these days I think.

For additional topics, as someone who has spent the past year learning Mithril, I would have appreciated documentation that gave examples/how to for these scenarios:

- Advanced routing for nested templates with an auth check
- Basic webpack config
- Centralizing state management, similar to Vuex
- Drag and drop
- Having all of the docs use ES6 module syntax :grin:
- Testing/test harnesses and stubs (add more details to ospec and maybe a section on Jest?)

Happy to add these somewhere.

**Comment by dead-claudia** [comment]
@thequailman 

> Having all of the docs use ES6 module syntax 😁

Already being considered separately in #1687 and #2224.
